cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/KafkaSetOperatorTest.java

```diff
@@ -134,7 +130,7 @@ public void testNeedsRollingUpdateEnvSomeOtherThing() {
         assertTrue(KafkaSetOperator.needsRollingUpdate(diff()));
     }
 
-    @Test
+    /*@Test
```

Comment:
this test is commented out, there is something more to do for having it running or just forgot to remove the comment block?

Reply:
As I said, the tests are not done for this. I wanted to get your comments to the code before writing the tests.

Reply:
Is it still the case that the tests are not done?

Reply:
Yeah, I forgot to delete this test :-). The reverting of the storage changes is not done here anymore - there are new tests in the model classes..